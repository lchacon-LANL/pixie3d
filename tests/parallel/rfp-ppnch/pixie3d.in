 &datin
    ilevel   = 0

    nxd      = 128
    nyd      = 32
    nzd      = 1

    xmax     = 1.
    ymax     = 0.
    zmax     = 1.

    numerical_grid = f

    bcs      = 'spt','def','per','per','per','per'

    eta      = 1d-3
    nu       = 1d-3
    dd       = 2d-3
    chi      = 0d-2
    
    equil    = 'ppnch'

    coords   = 'hel'

    dlambda  = 0.5
    gparams  = 1d0,-2.0d0,4d0,2d0,0d0

    gamma    = 1d0

    dt       = 5d-3
    tmax     = 2.5d-1
    dstep    = 2.5d-1

    pvx      = 1d-4
    odd      = t
    random   = f

    iguess   = 0

    cnfactor = 0.5

    restart  = f
    timecorr = f

    source   = t

    precon   = 'id'

    nc_eom_f = t
    nc_eom_v = f
    solve_rho= f

 /

 &graphdef
    E0 =  0.000000 ,  0.000000 ,  4.000000E-03
    sel_diag  = 2,12,15,17,18,19,20,22,23
    sel_graph = 1,2,3,5,6,7,9,10,11
    dplot = 1d-2
    prof_conf = 1,'r',1,20,1
    cont_conf = 3,'x','y',1,1,1
    hdf_plot = f
    clean    = f
 /

