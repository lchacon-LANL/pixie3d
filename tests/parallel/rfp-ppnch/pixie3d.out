 Calculating equilibrium ppnch...
 
 Equilibrium converged in  10 iterations with error =  0.19E-05
 
 Force free error=  4.304117187015763E-005
 Ohms law   error=  8.247988233499633E-005
 
 Pinch equilibrium features:
   Theta  =  1.64824680760764
   Bz(r=0)=  1.00000000000000
   I0     =  2.05448122853488
   E0/eta =  4.00000000000000
   T_flux =  0.623232279004647
 
 ***********************************************
 ****** PIXIE3D Extended MHD code, V. 0.9 ******
 ******       Developer: L. Chacon        ******
 ******  Los Alamos National Laboratory   ******
 ******       Los Alamos, NM 87545        ******
 ******           LA-CC 07-005            ******
 ***********************************************
 
 ****************** NOTICES ******************
 Using MAGNETIC FIELD version
 Using non-conservative JxB     part of EOM
 Using non-conservative grad(p) part of EOM
 Not solving for density: rho = 1
 ****************** NOTICES ******************

  Grid mesh:................  128x   32x    1
  Number of grid levels:....    5,    4,    1
  Preconditioning method:... id        
  Time correction...........   F
  External source...........   T
  Numerical grid............   F
  Advection scheme..........   ZIP    
  TS scheme................. Theta    
 
   itime    time     dt    vx_max vy_max vz_max GMRES  Newton  CN factor
      1    0.005  5.0E-03   0.012  0.000  0.000     3     3     0.500
      2    0.010  5.0E-03   0.012  0.000  0.000     3     3     0.500
      3    0.015  5.0E-03   0.012  0.000  0.000     4     3     0.500
      4    0.020  5.0E-03   0.012  0.000  0.000     7     5     0.500
      5    0.025  5.0E-03   0.012  0.000  0.000    10     5     0.500
      6    0.030  5.0E-03   0.012  0.000  0.000    10     5     0.500
      7    0.035  5.0E-03   0.012  0.000  0.000    15     5     0.500
      8    0.040  5.0E-03   0.012  0.000  0.000    11     5     0.500
      9    0.045  5.0E-03   0.012  0.000  0.000    14     5     0.500
     10    0.050  5.0E-03   0.012  0.000  0.000    14     5     0.500

Final statistics

 itime  Newt/ndt  GMRES/ndt  GMRES/Newt
   10       4.4        9.1        2.1
